<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QMainWindow" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>UserName</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rewrite Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_Username">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;this User name is used &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Error_Password">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Error, password not match&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_Age">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Age must be more than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_Check">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Not all data filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="Repassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="Date_of_birth">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Date of Birth </string>
    </property>
    <widget class="QComboBox" name="Day">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Month">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>82</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Jan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Feb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Apr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jun</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sep</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oct</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nov</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dec</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="year">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gender">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Gender</string>
    </property>
    <widget class="QRadioButton" name="female">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="male">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="account_type">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Account Type</string>
    </property>
    <widget class="QRadioButton" name="user_account">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="admin_account">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="genres">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Fav. Genres</string>
    </property>
    <widget class="QCheckBox" name="action">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="drama">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Drama</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="comedy">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Comedy</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="horror">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Horror</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="other">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="anime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Anime</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
